src/plugins/home/public/application/components/tutorial/instruction_set.js

```diff
@@ -278,7 +277,9 @@ class InstructionSetUi extends React.Component {
     return (
       <EuiSplitPanel.Outer>
         <EuiSplitPanel.Inner color="subdued" paddingSize="none">
-          <EuiTabs css={{ padding: `0 ${euiThemeVars.euiSizeL}` }}>{this.renderTabs()}</EuiTabs>
+          <EuiTabs css={({ euiTheme }) => ({ padding: `0 ${euiTheme.size.l}` })}>
+            {this.renderTabs()}
+          </EuiTabs>
```

Comment:
Accessing this element in Kibana:

https://github.com/user-attachments/assets/b420031b-ef4e-4bd3-8eb3-e41c9f9cc85c

Reply:
I might strip this change out of this PR because the Jest test that is impacted is pretty difficult to get updated.